.sonarqube/sonar-scanner.properties

```diff
@@ -14,101 +14,47 @@
 
 # Configuration file for Sonar Scanner used for CI 
 
-
-sonar.projectKey=usdot-fhwa-stol_autoware.auto
+sonar.projectKey=usdot-fhwa-stol_autoware-auto
 sonar.organization=usdot-fhwa-stol
 sonar.cfamily.cache.enabled=false
 sonar.cfamily.compile-commands=/opt/carma/build/compile_commands.json
 sonar.cfamily.gcov.reportsPath=/opt/carma/coverage_reports/gcov
+sonar.projectName=autoware.auto
+sonar.projectVersion=1.0
 sonar.host.url=https://sonarcloud.io
 # Set Git as SCM sensor
 sonar.scm.disabled=false
 sonar.scm.enabled=true
 sonar.scm.provider=git
-# This is the name and version displayed in the SonarCloud UI.
-sonar.projectName=autoware.auto
-sonar.projectVersion=1.0
-sonar.cfamily.gcov.reportsPath=/opt/carma/coverage_reports/gcov
-sonar.coverageReportPaths=/opt/carma/coverage_reports/gcov/coverage.xml
```

Comment:
I think we need this?
`sonar.coverageReportPaths=/opt/carma/coverage_reports/gcov/coverage.xml`